File: VB/ShowFilterPopupCheckedListBox example/Form1.vb
Imports System
Imports System.Windows.Forms
Imports DevExpress.XtraTreeList

Namespace ShowFilterPopupCheckedListBox_example

    Public Partial Class Form1
        Inherits Form

        Public Sub New()
            InitializeComponent()
        End Sub

        Private Sub Form1_Load(ByVal sender As Object, ByVal e As EventArgs)
            table1TableAdapter.Fill(treelistdbDataSet.Table1)
        End Sub

        Private Sub treeList1_ShowFilterPopupCheckedListBox(ByVal sender As Object, ByVal e As FilterPopupCheckedListBoxEventArgs)
            If Not Equals(e.Column.FieldName, "Department") Then Return
            ' disable "(Select All)" item
            e.CheckedComboBox.SelectAllItemVisible = False
        End Sub
    End Class
End Namespace

File: CS/ShowFilterPopupCheckedListBox example/Form1.cs
using System;
using System.Windows.Forms;
using DevExpress.XtraTreeList;

namespace ShowFilterPopupCheckedListBox_example
{
    public partial class Form1 : Form
    {
        public Form1()
        {
            InitializeComponent();
        }

        private void Form1_Load(object sender, EventArgs e)
        {
            this.table1TableAdapter.Fill(this.treelistdbDataSet.Table1);
        }

        private void treeList1_ShowFilterPopupCheckedListBox(object sender, FilterPopupCheckedListBoxEventArgs e)
        {
            if (e.Column.FieldName != "Department") return;
            // disable "(Select All)" item
            e.CheckedComboBox.SelectAllItemVisible = false;
        }
    }
}
